deployment/postgres-init-db/sql/default_user_roles.sql

```diff
@@ -38,6 +38,7 @@ set action_permissions = '{
       "get_conflicting_activities": "NO_CHECK",
       "get_non_conflicting_activities": "NO_CHECK",
       "get_plan_history": "NO_CHECK",
+      "migrate_plan_to_model": "PLAN_OWNER_COLLABORATOR",
```

Comment:
MAJOR, BLOCKING: This change was not in the migration. You can address it by adding the following line:

In the `up.sql`
```sql
update permissions.user_role_permission
  set function_permissions = function_permissions
                               || jsonb_build_object('migrate_plan_to_model', 'PLAN_OWNER_COLLABORATOR')
where role = 'user';
```

In the `down.sql`
```sql
update metadata.user_role_permission
  set function_permissions = function_permissions  - 'migrate_plan_to_model';
```

Note that in the `down`, we don't provide a `where` clause, as we want to remove the key from any role that has it assigned, while in the `up`, we only want to update our `user` role while not touching the client's custom roles/`aerie_admin`/`viewer`